pages/teams/[team].js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useRouter } from "next/router";
+import teams from "../../src/lib/teams";
+import Layout from "../../src/components/Layout";
+import NewsCard from "../../src/components/NewsCard";
+import TeamsList from "../../src/components/TeamsList";
+
+export default function Home({ data }) {
+  const router = useRouter();
+
+  return (
+    <Layout>
+      <div className="row">
+        <dd className="col-sm-3">
+          <h5>Produits</h5>
+          <TeamsList />
+        </dd>
+        <dd className="col-sm-9">
+          <h5>{(teams.getById(router.query.team) || {}).name || ""}</h5>
+          {data &&
+            data.map &&
+            data.map((d, i) => <NewsCard data={d} key={d.content.team + i} />)}
+        </dd>
+      </div>
+    </Layout>
+  );
+}
+
+Home.getInitialProps = async ({ req, query }) => {
+  let url = `/api/fetchTeamPosts?team=${query.team}`;
+
+  if (req) {
+    const protocol = req.headers["x-forwarded-proto"] || "http";
+    url = `${protocol}://${req.headers.host}${url}`;
+  }
+
+  const payload = await fetch(url);
+  const data = await payload.json();
+  data.sort(
+    (a, b) =>
+      new Date(b.content.date).getTime() - new Date(a.content.date).getTime()
+  );
+
+  return { data };
```

Comment:
you can pass `team: query.team` here so you dont need to require router in the next component

Reply:
Yep, true... since I get it in the `getInitialProps`.